content/en/05-reference/01-admin/09-user-mgmt.md

```diff
@@ -0,0 +1,107 @@
+</Alert>
+
+Pixie has features to share and manage access your Pixie organization:
+
+- [Invite users](/reference/admin/user-mgmt#invite-users)
+- [Enable approvals for new users](/reference/admin/user-mgmt#enable-approvals-for-new-users)
+- [Disable existing invite links](/reference/admin/user-mgmt#disable-invite-links)
+- [Remove users](/reference/admin/user-mgmt#remove-users)
+
+Users added to your organization can view Pixie live views, query running clusters, and deploy new Pixie clusters.
+
+## Invite Users
+
+### GSuite Organizations
+
+Pixie handles GSuite orgs differently than other orgs.
+
+You are part of a Pixie GSuite org if:
+
+1. The original creator of the org signs into Pixie using the "Sign in with Google" button AND
+2. The email the admin uses to sign in is part of a GSuite account.
+
+GSuite orgs can invite two types of users:
+
+- **Teammates with GSuite emails**: should use the "Sign-Up with Google" button on the [Sign up](https://work.withpixie.ai/auth/signup) page. GSuite emails will automatically join the Pixie org corresponding to their GSuite domain. _DO NOT use Invite Links to invite teammates with GSuite emails._ For greater control, enable [User Approvals](/reference/admin/user-mgmt#enable-approvals-for-new-users).
+
+- **Teammates using Gmail or creating custom email/password Pixie accounts**: should be sent [Invite Links](/reference/admin/user-mgmt#invite-links).
+
+### All Other Organizations
+
+If your organization is not a [GSuite organization](/reference/admin/user-mgmt#gsuite-organnizations), you can invite users with [Invite Links](/reference/admin/user-mgmt#invite-links).
+
+<Alert variant="outlined" severity="warning">Invited teammates using GSuite emails MUST sign up for Pixie using the "Sign-Up with Email" button. If a teammate accidentally signs up for Pixie using SSO ("Sign-Up with Google" button), they will not be able to join your Org. To fix this, they will need to sign up for Pixie again using the "Sign-Up with Email" button with a new or modified email (e.g. adding `+pixie` to their email, like anna+pixie@fakeemail.com)</Alert>
+
+### Invite Links
+
+1. Open the [Live UI](/using-pixie/using-live-ui) Admin page (`/admin`).
+2. Select the Users tab (`/admin/org`).
+
+<svg title='' src='admin/user-mgmt/users-tab-invite-button.png'/>
+
+3. Click the “+ Invite Users” button in the top right corner. This will pop open a modal with a generated invite link for your organization.
+
+<svg title='' src='admin/user-mgmt/invite-link.png'/>
+
+4. Copy the link and send it to your teammates.
+
+### Enable Approvals for New Users
+
+Combine Invite Links with User Approvals for greater control over who joins your organization. If your organization enables User Approvals, every user must be manually approved after they join your organization before they are able use Pixie.
```

Comment:
> Combine Invite Links with User Approvals for greater control over who joins your organization.

Slightly confused here - Invite links are NOT for GSuite users. But I thought user approvals could be. If user approval are handled differently from gsuite to username/password users, then we may want to just have a section for each type of account, and each subsection (Such as enable approvals) repeated within each section.

Reply:
IIUC, User Approvals also applies to Gsuite users (who join orgs automatically). I think the confusion could be resolved if I just delete the first line here. @vihangm

Reply:
GSuite based orgs can also invite users (the non GSuite kind). Invites scoping is at the org level so is user approval which is why it was phrased this way.
You are right that approvals will gate all users. Invited or auto added using GSuite. This was just written in a way to suggest that you have extra control over invite links by leveraging Approvals.
Feel free to reword.